packages/file-input/_file.scss

```
@@ -52,11 +57,11 @@
         box-shadow: 0 0 5px 0 rgba(0, 0, 0, 0.05);
 
         &--error {
-            background-color: colors.varslingsfarge("feil");
+            background-color: var(--jkl-error);
```

Merker at vi må prioritere punktet om dokumentasjon av variabler 😅 Er ikke så lett å finne frem i jungelen nå.

```suggestion
            background-color: var(--jkl-color-background-alert-error);
```